File: generateHashtag.js
function generateHashtag(str) {
    if (str.trim().length == 0) {
        console.log(false)
    } else {
        let wordsList = str.toLowerCase().split(' ');
        let finalWord = '';
        for (let i = 0; i < wordsList.length; i++) {
            finalWord = finalWord + (wordsList[i].charAt(0).toUpperCase() + wordsList[i].substring(1));
        }
        if (finalWord.length < 140) {
            console.log('#' + finalWord)
        } else {
            console.log(false)
        }
    }
}

generateHashtag("")
generateHashtag(" ".repeat(200))
generateHashtag("Do We have A Hashtag")
generateHashtag("Codewars")
generateHashtag("Codewars Is Nice")
generateHashtag("Codewars is nice")
generateHashtag("code" + " ".repeat(140) + "wars")
generateHashtag("Looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong Cat")
generateHashtag("a".repeat(139))
generateHashtag("a".repeat(140))

File: generate_hashtag.py
def generate_hashtag(s):
    if len(s) > 140 or len(s) == 0:
        return False
    else:
        return ''.join([word if i > 0 else '#'+word for i,
                        word in enumerate(s.title().split(' '))])
    # your code here


generate_hashtag('')
generate_hashtag('Do We have A Hashtag')[0]
generate_hashtag('Codewars')
generate_hashtag('Codewars      ')
generate_hashtag('Codewars Is Nice')
generate_hashtag('codewars is nice')
generate_hashtag('CodeWars is nice')
generate_hashtag('c i n')
generate_hashtag('codewars  is  nice')
generate_hashtag('Looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong Cat')
